admin/spam/views.py

```diff
@@ -174,6 +174,8 @@ def form_valid(self, form):
         try:
             if int(form.cleaned_data.get('confirm')) == Comment.SPAM:
                 item.confirm_spam(save=True)
+                item.is_deleted = True
```

Comment:
Is there ever a case where we would want to `confirm_spam` and not delete the comment?